test/cpp/jit/test_custom_class_registrations.cpp

```diff
@@ -33,6 +33,27 @@ struct Foo : torch::CustomClassHolder {
   }
 };
 
+struct FooGetterSetter : torch::CustomClassHolder {
+  int64_t x, y;
+  FooGetterSetter() : x(0), y(0) {}
+  FooGetterSetter(int64_t x_, int64_t y_) : x(x_), y(y_) {}
+
+  int64_t getX() {
+    return x;
+  }
+  void setX(int64_t z) {
+    x = z;
+  }
+
+  int64_t getY() {
+    return y;
+  }
```

Comment:
I would make these more complicated just to ensure that the tests aren't passing because of the getters actually being attribute lookups.